### ISSUE TITLE
save_trt.py 

### ISSUE THREAD

**Issue by eyebies** [huyhoang17/DB_text_minimal#25]
Thanks for sharing the code !Would it be possible to upload the save_trt file as well ?
thanks!

**Comment by huyhoang17** [comment]
Hi @eyebies ,
Currently, I don't have this script anymore. But you can convert this DB model to TensorRT format by using [trtexec](https://docs.nvidia.com/deeplearning/tensorrt/developer-guide/index.html#dla_topic)

Note, if used `F.interpolate` function, you must change the mode to "bilinear" instead of the current default mode "nearest" to avoid wrong output mask from Trt model (even though the model converts successfully)!
